packages/cascara/src/modules/ModuleError/ModuleError.js

```diff
@@ -16,7 +20,7 @@ const ModuleError = ({ moduleName, moduleOptions }) => {
   console.error(message);
 
   return (
-    <div className={styles.Error} data-testid={'module-error'}>
+    <div className={cx(className, 'Error')} data-testid={'module-error'}>
```

Comment:
Rocio, let's go through all of these and make sure our external defined className is the last argument for the `cx()` function.

Reply:
the `className` should be last, and it should also come from `rest.className` so we do not have to define it in our propTypes.